osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -171,7 +174,7 @@ public Catcher([NotNull] Container trailsTarget, BeatmapDifficulty difficulty =
         [BackgroundDependencyLoader]
         private void load(OsuConfigManager config)
         {
-            hitLighting = config.GetBindable<bool>(OsuSetting.HitLighting);
```

Comment:
This is bad as `GenerateHitLighting` set in initializer is silently overwritten. Probably this logic should be moved to `CatchPlayfield`.